modules/getting-started/proc-customize-rhdh-tech-radar-page.adoc

```diff
@@ -21,33 +13,22 @@ You can provide data to the Tech Radar page from the following sources:
 
 .Prerequisites
 
-You have installed {product} by using either the Operator or Helm chart.
-For more information, see xref:{installing-on-ocp-book-url}#assembly-install-rhdh-ocp[{installing-on-ocp-book-title}].
+* You have installed {product} by using either the Operator or Helm chart. For more information, see link:{installing-on-ocp-book-url}[{installing-on-ocp-book-title}].
+* You have specified the data sources for the Tech Radar plugin in the `integrations` section of the `app-config.yaml` file. For more information, see link:{authentication-book-url}#authenticating-with-github[Authenticating with GitHub].
```

Comment:
Do you think this might be better?
```suggestion
* You have specified the data sources for the Tech Radar plugin in the `integrations` section of the `app-config.yaml` file. For example, to configure GitHub as a data source, see link:{authentication-book-url}#authenticating-with-github[Configuring GitHub Integration].
```